packages/solana/src/connectors/walletConnectConnector.ts

```diff
@@ -1,20 +1,20 @@
 import base58 from 'bs58'
-import { PublicKey, Transaction, VersionedTransaction, type SendOptions } from '@solana/web3.js'
+import { Connection, Transaction, VersionedTransaction, type SendOptions } from '@solana/web3.js'
 import { OptionsController } from '@web3modal/core'
 
 import { SolStoreUtil } from '../utils/scaffold/index.js'
 import { UniversalProviderFactory } from './universalProvider.js'
-import { BaseConnector } from './baseConnector.js'
+import { BaseConnector, type Connector } from './baseConnector.js'
```

Comment:
split regular imports and type imports:
```suggestion
import { BaseConnector } from './baseConnector.js'
import type { Connector } from './baseConnector.js'
```